build.go

```diff
@@ -167,20 +167,41 @@ func (c *Client) validateImageReference(imageName string) (name.Reference, error
 	return ref, nil
 }
 
+func mapAppDirError(dir string, err error) error {
```

Comment:
Pulling out the error logic feels a little awkward to me. I think we could just return the raw error from processAppDir even in the NotExist case. The os.NotExist error will contain the path that we are writing in this custom error, making it seem unnecessary.

Reply:
Yeah. I did this prior to the refactor when the error wasn't already prefixed with the path. 

After making the suggested change this is what it looks like:

```
ERROR: invalid app dir '/tmp/does-not-exists': lstat /private/tmp/does-not-exists: no such file or directory
```